README.md

```diff
@@ -27,7 +27,7 @@ Modo2Auth auth = new Modo2Auth(apiSecret, apiId);
 
 # 2 - Define the Request URL and Request body
 String apiUri = "/v2/reports";
-String fullUri = "https://checkout.mtktest.modopayments.net/v2/reports";
+String fullUri = "https://checkout.prod.modopayments.net/v3/checkout/list";
```

Comment:
We need to use [URL_HERE] since this endpoint doesn't exist. Nothing on prod uses v3 and that will prevent any confusion.

Reply:
or alternatively, use localhost:82 and add a comment about using the correct URL